tests/unit/src/test/resources/definition-scala3/example/StructuralTypes.scala

```diff
@@ -9,12 +9,12 @@ object StructuralTypes/*StructuralTypes.scala*/:
   }
 
   val user/*StructuralTypes.scala*/ = null.asInstanceOf[User/*StructuralTypes.scala*/]
-  user/*StructuralTypes.scala*/.name/*<no symbol>*/
-  user/*StructuralTypes.scala*/.age/*<no symbol>*/
+  user/*StructuralTypes.scala*/./*unexpected: scala.reflect.Selectable#selectDynamic().*/name/*Selectable.scala*/
```

Comment:
We should probably fix in in a follow up https://github.com/scalameta/metals/issues/5621

Reply:
Where does the `unexpected:` info come from?

Reply:
I think this is added in the test, probably an additional condition like if the name doesn't match